README.md

```diff
@@ -57,30 +58,48 @@ and deletes the release.
-| `image.pullPolicy`          | Image pull policy                                         | `Always`        |
-| `image.pullSecret`          | Specify image pull secrets                                | `nil`           |
-| `pod.annotations`           | Specify annotations for the pod                           | `nil`           |
-| `service.type`              | Specify the service type - only LB and NodePort supported | `LoadBalancer`  |
-| `service.nodePort`          | Specify the service nodePort                              | `30987`         |
-| `resources`                 | Kubernetes resources                                      | `nil`           |
-| `nodeSelector`              | Kubernetes nodeSelectors for the deployment               | `nil`           |
-| `tolerations`               | Kubernetes tolerations for the deployment                 | `nil`           |
-| `affinity`                  | Kubernetes affinities for the deployment                  | `nil`           |
-| `persistence.enabled`       | Enable/Disable persistence                                | `disabled`      |
-| `persistence.accessMode`    | Accessmode for the PVC                                    | `nil`           |
-| `persistence.existingClaim` | Name of an existing PVC to use                            | `nil`           |
-| `persistence.annotations`   | Annotations for the PVC                                   | `nil`           |
-| `persistence.storageClass`  | StorageClass for the PVC                                  | `nil`           |
-| `persistence.storageSize`   | Size of the PVC                                           | `nil`           |
+| Parameter                   | Description                                                                                 | Default         |
+|-----------------------------|---------------------------------------------------------------------------------------------|-----------------|
+| `image.repository`          | Image repository                                                                            | `teamspeak`     |
+| `image.tag`                 | Image tag                                                                                   | `3.12.0`        |
+| `image.pullPolicy`          | Image pull policy                                                                           | `Always`        |
+| `image.pullSecret`          | Specify image pull secrets                                                                  | `nil`           |
+| `pod.annotations`           | Specify annotations for the pod                                                             | `nil`           |
+| `service.type`              | Specify the service type - only LB and NodePort supported                                   | `LoadBalancer`  |
+| `service.ip`                | Specify the requested IP Address - only LB supported                                        | `nil`           |
```

Comment:
Not strictly necessary but left in after I realised MetalLB will auto assign both services the same IP.

Useful on its own though.